docs/content/docs/troubleshooting/traffic/ingress.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Ingress Troubleshooting"
+description: "Ingress Troubleshooting Guide"
+type: docs
+aliases: ["ingress.md"]
+---
+
+## When Ingress is not working as expected
+
+### 1. If using HTTPS ingress, confirm HTTPS ingress is enabled on OSM configmap
```

Comment:
Could you rename this section to:
`Confirm global ingress configuration is set as expected`, and document how to check whether the global setting is `HTTP` or `HTTPS`.